react/css/example/index.css

```diff
@@ -13382,6 +13382,50 @@ button:hover {
 .popup-header-fix {
   margin-top: -0.5rem !important; }
 
+.selecting-boundary-div {
```

Comment:
pls ensure you don't update any existing classes which may cause issues for others